### ISSUE TITLE
Warrior Spell: Heroic Leap wrong spell ID#

### ISSUE THREAD

**Issue by RickStrife** [TrinityCore/TrinityCore#28997]
### Description

Warrior's Heroic Leap Spell has no effect or animation when used. In this case the audio effects are triggered, the spell goes into cooldown, but the animation does not work, meaning the warrior stays in place.

The spell actually works, is coded in src --> server --> scripts --> Spells --> spell_warrior.cpp

It's declared in line 51, but has the wrong spell ID for the whole animation and movement.

enum WarriorSpells
{
SPELL_WARRIOR_HEROIC_LEAP_JUMP                  = 178368, -->> This is the wrong spell ID

it should be:

SPELL_WARRIOR_HEROIC_LEAP_JUMP                  = 94954

As you can see here: https://www.wowhead.com/spell=94954/dnd-cosmetic-heroic-leap-dest

When you change the spell ID and re-compile the core, The spell works normally, full animation and everything.

### Expected behaviour

When you use the warrior's heroic leap spell, you select your destination area and press it in order for the warrior to jump, travel the selected distance, and it should land in the selected area.

### Steps to reproduce the problem

1. Select warrior's heroic leap, then select the distance or destination, and click on it.
2. Expect your character to land on the selected destination.
3. Nothing happens. 
4. Go to and edit src --> server --> scripts --> Spells --> spell_warrior.cpp and change **SPELL_WARRIOR_HEROIC_LEAP_JUMP                  = 178368**, to  **SPELL_WARRIOR_HEROIC_LEAP_JUMP                  = 94954** Then save and recompile the core.
5. Try using Heroic Leap again, and this time it works! The Warrior jumps and lands onto the selected destination.

### Branch

master

### TC rev. hash/commit

TrinityCore rev. 33bd04fb6495+ 2023-05-16 11:35:20 +0200 (master branch) (Win64, RelWithDebInfo, Static) (worldserver-daemon)

### Operating system

Windows 11

### Custom changes

None

**Comment by Shauren** [comment]
No, spells are not fixed by randomly finding a similar named spell that does similar things. Heroic Leap does not need a spellscript at all

**Comment by RickStrife** [comment]
Well, I understand, but changing the spell ID works for me. The moment I change it, It starts working. The spell function itself works perfectly, the problem is the ID in relation to the spell script for the actual movement and animation of the jump to show on the screen. I have tested it.

**Comment by Zaekor** [comment]
Indeed this is not the right way to solve this problem.

I will try to find a fix
